kalite/distributed/templates/distributed/base.html

```diff
@@ -158,7 +158,7 @@
           CENTRAL_SERVER_HOST         : "{{ central_server_host }}",
           SECURESYNC_PROTOCOL         : "{{ securesync_protocol }}",
           CURRENT_LANGUAGE            : "{{ current_language }}",
-          CHANNEL                     : "{{ channel }}",
+          CHANNEL                     : "{{ channel }}"
```

Comment:
We don't support IE 7 _or_ 8, due to some incompatibility with an external JS library. ([It's PDF.js](https://github.com/mozilla/pdf.js/wiki/Frequently-Asked-Questions#faq-support).)